README.md

```diff
@@ -477,7 +477,7 @@ pytest -k "not slow"
 Each experiment should be tagged in order to easily filter them across files or in logger UI:
 
 ```bash
-python train.py tags=["mnist", "experiment_X"]
+python src/train.py tags=["mnist","experiment_X"]
```

Comment:
```suggestion
python train.py tags=["mnist","experiment_X"]
```
Is there a reason to changing it to `src/...` ? Other examples in readme use only `python train.py ...`